### PR TITLE
fix: indent attribute doc

### LINKED ISSUE #30: wrong indenting when generate from schema/attribute description to docstring
## Bug Report

when translating k8s openapi to KCL models, the attribute description may contain line break 

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
1. save following code as `oai.json`
```
{
   "definitions": {
      "io.k8s.api.admissionregistration.v1.MutatingWebhook": {
         "description": "MutatingWebhook describes an admission webhook and the resources and operations it applies to.",
         "properties": {
            "matchPolicy": {
               "description": "matchPolicy defines how the \"rules\" list is used to match incoming requests. Allowed values are \"Exact\" or \"Equivalent\".\n\n- Exact: match a request only if it exactly matches a specified rule. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, but \"rules\" only included `apiGroups:[\"apps\"], apiVersions:[\"v1\"], resources: [\"deployments\"]`, a request to apps/v1beta1 or extensions/v1beta1 would not be sent to the webhook.\n\n- Equivalent: match a request if modifies a resource listed in rules, even via another API group or version. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, and \"rules\" only included `apiGroups:[\"apps\"], apiVersions:[\"v1\"], resources: [\"deployments\"]`, a request to apps/v1beta1 or extensions/v1beta1 would be converted to apps/v1 and sent to the webhook.\n\nDefaults to \"Equivalent\"",
               "type": "string"
            }
         }
      }
   },
   "info": {
      "title": "Kubernetes",
      "version": "unversioned"
   },
   "paths": {},
   "swagger": "2.0"
}
```
2. run `kcl-openapi generate model -f oai.json -t output`
3. run `cat output/models/io_k8s_api_admissionregistration_v1_mutating_webhook.k`

### 2. What did you expect to see? (Required)

```
"""
This file was generated by the KCL auto-gen tool. DO NOT EDIT.
Editing this file might prove futile when you re-run the KCL auto-gen generate command.
"""


schema MutatingWebhook:
    """ MutatingWebhook describes an admission webhook and the resources and operations it applies to.

    Attributes
    ----------
    matchPolicy : str, default is Undefined, optional
        matchPolicy defines how the "rules" list is used to match incoming requests. Allowed values are "Exact" or "Equivalent".
 
        - Exact: match a request only if it exactly matches a specified rule. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, but "rules" only included `apiGroups:["apps"], apiVersions:["v1"], resources: ["deployments"]`, a request to apps/v1beta1 or extensions/v1beta1 would not be sent to the webhook.
 
        - Equivalent: match a request if modifies a resource listed in rules, even via another API group or version. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, and "rules" only included `apiGroups:["apps"], apiVersions:["v1"], resources: ["deployments"]`, a request to apps/v1beta1 or extensions/v1beta1 would be converted to apps/v1 and sent to the webhook.
        Defaults to "Equivalent"
    """


    matchPolicy?: str



```

### 3. What did you see instead (Required)

```
"""
This file was generated by the KCL auto-gen tool. DO NOT EDIT.
Editing this file might prove futile when you re-run the KCL auto-gen generate command.
"""


schema MutatingWebhook:
    """ MutatingWebhook describes an admission webhook and the resources and operations it applies to.

    Attributes
    ----------
    matchPolicy : str, default is Undefined, optional
         matchPolicy defines how the "rules" list is used to match incoming requests. Allowed values are "Exact" or "Equivalent".
 
 - Exact: match a request only if it exactly matches a specified rule. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, but "rules" only included `apiGroups:["apps"], apiVersions:["v1"], resources: ["deployments"]`, a request to apps/v1beta1 or extensions/v1beta1 would not be sent to the webhook.
 
 - Equivalent: match a request if modifies a resource listed in rules, even via another API group or version. For example, if deployments can be modified via apps/v1, apps/v1beta1, and extensions/v1beta1, and "rules" only included `apiGroups:["apps"], apiVersions:["v1"], resources: ["deployments"]`, a request to apps/v1beta1 or extensions/v1beta1 would be converted to apps/v1 and sent to the webhook.
 Defaults to "Equivalent"
    """


    matchPolicy?: str



```
### 4. What is your KusionStack components version? (Required)
--

### PR DESCRIPTION
**Pull request by amyXia1994**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [X] Y 

fix #30 

#### 2. What is the scope of this PR (e.g. component or file name):

pkg/swagger/generator/template_repo.go
pkg/swagger/generator/templates/header.gotmpl
pkg/swagger/generator/templates/introduction.gotmpl

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

fix the indentation in generated attribute doc

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [X] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [X] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

pkg/swagger/generator/template_repo_test.go

#### 6. Release note

Please refer to [Release Notes Language Style Guide](https://kusionstack.io/docs/governance/release-policy/) to write a quality release note.

```release-note
fix the indentation in generated attribute doc
```

### CODE REVIEW

File: pkg/swagger/generator/template_repo.go [hunk 1]
```diff
@@ -487,6 +488,21 @@ func padSurround(entry, padWith string, i, ln int) string {
 	return strings.Join(res, ",")
 }
 
+func padDocument(str string, pad string) string {
+	// indent multi line document with given pad
+	lines := strings.Split(str, "\n")
```
Comment (Peefy) [inline pkg/swagger/generator/template_repo.go:493]: Considering the Windows newline `\r\n`

Comment (amyXia1994) [reply pkg/swagger/generator/template_repo.go]: > Considering the Windows newline `\r\n`

fixed. ptal

File: pkg/swagger/generator/template_repo.go [hunk 2]
```diff
@@ -487,6 +489,29 @@ func padSurround(entry, padWith string, i, ln int) string {
 	return strings.Join(res, ",")
 }
 
+// padDocument indent multi line document with given pad
+func padDocument(str string, pad string) string {
+	// get the OS name
+	// set the appropriate line separator
+	var lineSeparator string
+	if runtime.GOOS == "windows" {
```
Comment (Peefy) [inline pkg/swagger/generator/template_repo.go:497]: Ref: https://github.com/open-policy-agent/conftest/blob/40d4f223b2052cc32766b46c0e6cbc8f3c54ac63/parser/yaml/yaml.go#L36

Comment (amyXia1994) [reply pkg/swagger/generator/template_repo.go]: fixed.

That will suit the cases when `\r\n`  are explicitly declared in the openapi spec file.

**Review by Peefy — APPROVED** [review]
LGTM

**Pull request merged**